src/api/public/notes/notes.controller.ts

```diff
@@ -188,16 +189,16 @@ export class NotesController {
   @UseGuards(TokenAuthGuard)
   @Get(':noteIdOrAlias/metadata')
   async getNoteMetadata(
-    @Request() req,
+    @Req() req: Request,
     @Param('noteIdOrAlias') noteIdOrAlias: string,
   ): Promise<NoteMetadataDto> {
     try {
       const note = await this.noteService.getNoteByIdOrAlias(noteIdOrAlias);
       if (!this.permissionsService.mayRead(req.user, note)) {
         throw new UnauthorizedException('Reading note denied!');
       }
-      return this.noteService.toNoteMetadataDto(note);
-    } catch (e) {
+      return await this.noteService.toNoteMetadataDto(note);
+    } catch (e: unknown) {
```

Comment:
Why is this added here? In other places there is no annotation.

Reply:
removed